libfoundation/src/foundation-string.cpp

```diff
@@ -1432,6 +1434,7 @@ bool MCStringMapIndices(MCStringRef self, MCBreakIteratorType p_type, MCLocaleRe
     if (t_start == kMCLocaleBreakIteratorDone)
     {
         r_out_range = MCRangeMake(MCStringGetLength(self), 0);
+		MCLocaleBreakIteratorRelease (t_iter);
```

Comment:
There is a general MCAuto class for things which have a 'FREE' method (which is what MCLocaleBreakIteratorRelease) is - MCAutoCustomPointer.